public/app/features/explore/ContentOutline/ContentOutline.tsx

```diff
@@ -60,7 +60,9 @@ export function ContentOutline({ scroller, panelId }: { scroller: HTMLElement |
 
     do {
       scrollValue += el?.offsetTop || 0;
-      el = el?.offsetParent as HTMLElement;
+      if (el?.offsetParent instanceof HTMLElement) {
```

Comment:
Hmm... now it could theoretically get into an infinite loop if offsetParent is not ant HTMLElement 🤔

Reply:
@ifrost ah good spot! 👀

previously if it hit something that wasn't an `HTMLElement` it would set `el = el?.offsetParent` which would be `undefined` and shortcircuit the `while` loop.

have pushed a change to handle this:

```tsx
    do {
      scrollValue += el?.offsetTop || 0;
      el = el?.offsetParent instanceof HTMLElement ? el.offsetParent : undefined;
    } while (el && el !== scroller);
```